apps/test/unit/lib/kits/maker/dropletConfigTest.js

```diff
@@ -81,3 +88,71 @@ describe('getBoardEventDropdownForParam', function () {
     });
   });
 });
+
+// TODO (bbuchanan): Replace with more general assertions when we move maker
+// commands back to this kit.
+describe(`timedLoop(ms, callback)`, function () {
```

Comment:
Some test helpers ([like `sinon.test()`](http://sinonjs.org/docs/#sandbox)) depend on being able to override `this` bindings.  I suspect that's not an issue for any of these tests, but I've habitually used regular function expressions since I learned that.